<!-- Copyright 2026 bitserv-ai -->
diff --git a/vllm/v1/attention/backends/rocm_aiter_fa.py b/vllm/v1/attention/backends/rocm_aiter_fa.py
index 7e850af3e..2386147a8 100644
--- a/vllm/v1/attention/backends/rocm_aiter_fa.py
+++ b/vllm/v1/attention/backends/rocm_aiter_fa.py
@@ -768,12 +768,12 @@ class AiterFlashAttentionBackend(AttentionBackend):
 
     @classmethod
     def supports_compute_capability(cls, capability: DeviceCapability) -> bool:
-        from vllm.platforms.rocm import on_mi3xx
+        from vllm.platforms.rocm import on_mi3xx, on_gfx1x
 
         # DeviceCapability is currently created using torch.cuda.get_device_capability()
         # which is known to be buggy on rocm systems. on_mi3xx uses amd-smi which is
         # more reliable.
-        return on_mi3xx()
+        return on_mi3xx() or on_gfx1x()
 
     @classmethod
     def supports_non_causal(cls) -> bool:
