<!-- Copyright 2026 bitserv-ai -->
diff --git a/vllm/_aiter_ops.py b/vllm/_aiter_ops.py
index 95a536103..299e8b59a 100644
--- a/vllm/_aiter_ops.py
+++ b/vllm/_aiter_ops.py
@@ -105,9 +105,9 @@ def is_aiter_found_and_supported() -> bool:
     VLLM_ROCM_USE_AITER=0, while preventing unwanted JIT warnings for auto-discovery.
     """
     if current_platform.is_rocm() and IS_AITER_FOUND:
-        from vllm.platforms.rocm import on_mi3xx
+        from vllm.platforms.rocm import on_mi3xx, on_gfx1x
 
-        return on_mi3xx()
+        return on_mi3xx() or on_gfx1x()
     return False
 
 
